.github/workflows/e2e_tests_lightspeed_evaluation.yaml

```
@@ -1,9 +1,7 @@
 name: E2E Tests for Lightspeed Evaluation
 
 on:
-  push:
-  pull_request_target:
-    types: [opened, synchronize, reopened, labeled]
+  []
```

actionlint (1.7.12)

[error] "on" section should not be empty

(syntax-check)